js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceContext.tsx

```diff
@@ -178,6 +183,7 @@ export const ROOT_WORKSPACE_QUERY = gql`
   ${PYTHON_ERROR_FRAGMENT}
   ${REPOSITORY_INFO_FRAGMENT}
   ${SENSOR_SWITCH_FRAGMENT}
+  ${BASIC_INSTIGATION_STATE_FRAGMENT}
```

Comment:
I'm not sure we should be adding more stuff to the root workspace query, since we're hoping to get rid of it? It seems like this PR is a nice improvement but moves in the opposite direction (more root workspace, with stronger caching)

Reply:
We're thinking of actually going completely in that other direction. Full root workspace query with indexedb caching, relying on the deployment status query to figure out when a workspace is stale or not and just automagically managing it for the app rather than having you needing to fetch information.

The reason I added these fields was to make the cached result usable by the sensors page in order to immediately use this data if available.